src/conversion.cpp

```diff
@@ -86,6 +87,21 @@ int32_t conversion::bytevector_to_int32(const types::ByteVector& bytevector) {
     return result;
 }
 
+float conversion::bytevector_to_float32(const types::ByteVector& bytevector) {
+    EVLOG(debug) << "Converting bytevector to float32: " << everest::connection::utils::get_bytes_hex_string(bytevector);
+
+    // Reversing bytevector since MODBUS is big-endian and ARM is little-endian
+    uint8_t inv_bytes[4];
+    for (int i = 0 ; i < 4 ; i++) inv_bytes[i] = bytevector[4-i-1];
```

Comment:
I would advise using curly brackets here as well for clarity

Reply:
Done.